.github/package-template.yml

```
@@ -97,8 +96,7 @@ jobs:
         shell: bash
         run: |
           artifact=$(find artifacts/${{matrix.alpine}} -type f -name \*.apk)
-          echo "::set-output name=path::$artifact"
-          echo "set output path=$artifact"
+          echo "path=$artifact" | tee -a $GITHUB_OUTPUT
```

Why use `tee -a` instead of this?

```suggestion
          echo "path=$artifact" >> $GITHUB_OUTPUT
```

This the recommended way from the examples in the blog post

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#examples

---

Because with `tee` we get to see the output in the logs